The job `dist-mips64-linux` of your PR [failed](https://github.com/rust-lang-ci/rust/runs/1032542684) ([pretty log](https://rust-lang.github.io/rust-log-analyzer/log-viewer/#https://github.com/rust-lang-ci/rust/commit/fe8b2dfab7b2bef915a81d82fd83979c5e197015/checks/1032542684/logs), [raw log](https://github.com/rust-lang-ci/rust/commit/fe8b2dfab7b2bef915a81d82fd83979c5e197015/checks/1032542684/logs)). Through arcane magic we have determined that the following fragments from the build log may contain information about the problem.

<details><summary><i>Click to expand the log.</i></summary>

```plain
   Compiling addr2line v0.13.0
[RUSTC-TIMING] addr2line test:false 0.362
[RUSTC-TIMING] gimli test:false 4.050
[RUSTC-TIMING] object test:false 7.975
error[E0658]: `#[doc(spotlight)]` is experimental
    |
    |
488 | #[doc(spotlight)]
    |
    |
    = note: see issue #45040 <https://github.com/rust-lang/rust/issues/45040> for more information
    = help: add `#![feature(doc_spotlight)]` to the crate attributes to enable

error[E0658]: `#[doc(spotlight)]` is experimental
     |
     |
1241 | #[doc(spotlight)]
     |
     |
     = note: see issue #45040 <https://github.com/rust-lang/rust/issues/45040> for more information
     = help: add `#![feature(doc_spotlight)]` to the crate attributes to enable
error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0658`.
[RUSTC-TIMING] std test:false 1.720
---
== clock drift check ==
  local time: Wed Aug 26 16:37:25 UTC 2020
  network time: Wed, 26 Aug 2020 16:37:25 GMT
== end clock drift check ==
##[error]Process completed with exit code 1.
Terminate orphan process: pid (2759) (node)
Terminate orphan process: pid (2787) (python)
```

</details><p></p>

[I'm a bot](https://github.com/rust-ops/rust-log-analyzer)! I can only do what humans tell me to, so if this was not helpful or you have suggestions for improvements, please ping or otherwise contact **`@rust-lang/infra`**. ([Feature Requests](https://github.com/rust-ops/rust-log-analyzer/issues?q=is%3Aopen+is%3Aissue+label%3Afeature-request))

--- a/library/std/src/lib.rs
+++ b/library/std/src/lib.rs
@@ -247,7 +247,7 @@
 #![feature(doc_cfg)]
 #![feature(doc_keyword)]
 #![feature(doc_masked)]
-#![cfg_attr(not(bootstrap), feature(doc_spotlight))]
+#![feature(doc_spotlight)]
 #![feature(dropck_eyepatch)]
 #![feature(duration_constants)]
 #![feature(exact_size_is_empty)]

